src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -818,6 +818,7 @@ const WorkflowView = _.flow(
       _.map(k => ({ name: k, inputType: 'unknown' }), _.keys(modifiedConfig[key])) :
       modifiedInputsOutputs[key]
     const filteredData = key === 'inputs' && !includeOptionalInputs ? _.reject('optional', data) : data
+    const isSingleOutputs = key === 'outputs' && this.isSingle()
```

Comment:
Nit: This name isn't intuitive to me. I don't have great suggestions. Seems like it might be more clear to inline it... which I also don't love. Do others have ideas?

Reply:
(from mob) consider renaming to isSingleAndOutputs